src/app/bidder/bidder.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="row">
+<div *ngIf="!bidderHistory" class="row">
   <div class="col-2" style="background-color: lightgrey">
       <h5>Filter by Tag:</h5>
```

Comment:
Shouldn't the bidder be able to filter the items in the history as well? Or are we saying no to that?